scripts/feature_tests/upgrade/controlplane_upgrade/3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh

```diff
@@ -14,27 +14,48 @@ start_logging "${1}"
 
 set_number_of_master_node_replicas 3
 set_number_of_worker_node_replicas 1
-
 provision_controlplane_node
+sleep 60
 
+get_target_kubeconfig
```

Comment:
Use 'set_kubeconfig_towards_target_cluster' when pointing to the target cluster the first time

Reply:
Done